language/types_values_variables.md

```diff
@@ -269,7 +274,7 @@ Represents the abstract notion of "value", its subtypes are `Numeric`, `String`
 
 ### Numeric
 
-Represents the abstract notion of "number", its subtypes are `Integer`, and `Float`.
+Represents the abstract notion of "number", its subtypes are `Integer`, `Float`, `Timespan` and `Timestamp`.
```

Comment:
The implementation is based on the fact that they are numeric and represents seconds. I'm not sure I see the problem in viewing them that way.